cli/azd/cmd/cmd_help.go

```diff
@@ -258,11 +258,18 @@ func getPreFooter(c *cobra.Command) string {
 }
 
 // generateCmdHelpDescription construct a help text block from a title and description notes.
-func generateCmdHelpDescription(title string, notes []string) string {
+func generateCmdHelpDescription(title string, notes []string, options ...string) string {
 	var note string
 	if len(notes) > 0 {
 		note = fmt.Sprintf("%s\n\n", strings.Join(notes, "\n"))
 	}
+	if len(options) > 0 {
+		var item string
+		for _, opt := range options {
+			item = opt
+		}
+		return fmt.Sprintf("%s\n\n%s%s\n\n", title, note, item)
+	}
 	return fmt.Sprintf("%s\n\n%s", title, note)
```

Comment:
I'm not sure I'm convinced we need this.  Based on the usage here could we not just add that extra line of text as part of the title and prints above the options?

Reply:
Agree.
And I think the entire text for the tittle needs to be reviewed now by @savannahostrowski  and @Austinauth .

We are mentioning the Azure sub and location are storage in the config... but what we storage is the default selections on the first time.  The actual selection is storage within each environment.

It might be useful to mention the alpha features here.

For the note about `AZD_CONFIG_DIR `, I would have it either as an extra bullet (within the notes) or as part of the title, as @wbreza  mentioned.

The structure of the help message was designed by @Austinauth as a title and notes, and all commands are following that structure.  Let's review with him first if we want to add this new section.

Reply:
I'd wonder if the title text should be everything up until the comma and then there should be a short sub-paragraph that details it stores things like sub/location per environment and alpha features (good call out Victor!)

Reply:
@vhvb1989 For `AZD_CONFIG_DIR`, I have update it as an extra bullet. Please review.